Library/Formula/letsencrypt.rb

```diff
@@ -0,0 +1,186 @@
+class Letsencrypt < Formula
+  desc "Tool to automatically receive and install X.509 certificates"
+  homepage "https://letsencrypt.org/"
+  url "https://github.com/letsencrypt/letsencrypt.git",
```

Comment:
They have a tagged release with a tarball available. Should we prefer that to fetching from the repo? https://github.com/letsencrypt/letsencrypt/releases/tag/v0.1.1-corrected